packages/compass-components/src/components/leafygreen.tsx

```diff
@@ -1,25 +1,23 @@
 import React from 'react';
```

Comment:
nothing urgent, but is this file still necessary?

Reply:
Not really necessary, no, this could all go in the `index.ts`. I think it might be nice to have this structure to keep that file a bit more readable and have a dedicated place for any future temporary wrappers we might need around LeafyGreen. Happy to change it if you feel strongly about where things should go.